docker-compose.yml

```diff
@@ -1,5 +1,3 @@
-version: '3.8'
```

Comment:
I think this is fine but not sure why it's part of the PR. Did you run into a local issue?